src/components/Cluster/NewCluster/AvailabilityZonesSelector.js

```diff
@@ -0,0 +1,83 @@
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+import React from 'react';
+import cmp from 'semver-compare';
+import NumberPicker from 'UI/NumberPicker';
+
+const FlexWrapperAZDiv = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+
+  p {
+    margin-right: 18px;
+    transform: translateY(-4px);
+  }
+`;
+
+const AvailabilityZonesSelector = ({
+  minValue,
+  maxValue,
+  value,
+  readOnly,
+  onChange,
+  requiredReleaseVersion,
+  currentReleaseVersion,
+}) => {
+  const isAvailable = cmp(currentReleaseVersion, requiredReleaseVersion) >= 0;
+
+  return (
+    <label className='availability-zones' htmlFor='availability-zones'>
+      <span className='label-span'>Availability Zones</span>
+      {isAvailable ? (
+        <FlexWrapperAZDiv>
+          <p>Number of availability zones to use:</p>
+          <div>
+            <NumberPicker
+              label=''
+              max={maxValue}
+              min={minValue}
+              onChange={onChange}
+              readOnly={readOnly}
+              stepSize={1}
+              value={value}
+            />
+          </div>
+        </FlexWrapperAZDiv>
+      ) : (
+        <>
+          <p>
+            Selection of availability zones is only possible for release version
+            {requiredReleaseVersion} or greater.
+          </p>
+          <div className='col-3'>
+            <NumberPicker readOnly={true} value={1} />
+          </div>
+        </>
+      )}
+    </label>
+  );
+};
+
+AvailabilityZonesSelector.defaultProps = {
+  minValue: 1,
+  maxValue: 1,
+  // eslint-disable-next-line no-empty-function
+  onChange: () => {},
+  readOnly: false,
+  value: 1,
+  requiredReleaseVersion: '0.0.1',
+  currentReleaseVersion: '0.0.2',
+};
+
+AvailabilityZonesSelector.propTypes = {
+  minValue: PropTypes.number,
+  maxValue: PropTypes.number,
+  onChange: PropTypes.func,
+  readOnly: PropTypes.bool,
+  value: PropTypes.number,
+  requiredReleaseVersion: PropTypes.string,
+  currentReleaseVersion: PropTypes.string,
+};
+
+export default AvailabilityZonesSelector;
```

Comment:
This component kind of overlaps with `AvailabilityZonesParser`, ¿Why you decided to wrote this one instead of changing that component to support this use case?

Reply:
I did not write this, this was taken out of the `CreateRegularCluster` component

Reply:
Anyway, a lot of content is duplicated. And probably we have too many `AvailabilityZonesXxxx` components.

I'm ok with leaving it like this if you think it's not worth to modify `AvailabilityZonesParser`, but at least we could change names to something like `AZSelectorV5/V4` to make things easier.

Reply:
I'll add an adapter to use `AvailabilityZonesParser` behind the scenes, and rename it to something like `V4AvailabilityZonesSelector`, to not over-complicate `AvailabilityZonesParser`